torch/_inductor/runtime/triton_heuristics.py

```diff
@@ -1156,6 +1155,8 @@ def benchmark_one_config(config):
             config2launcher[best_config] = self._precompile_config(
                 best_config
             ).make_launcher()
+
+        log.error(f"Function hash {generate_lookup_hash_from_source_code(str(self.size_hints), self.fn.src)} has best config {best_config}")
```

Comment:
```suggestion
        log.error("Function hash %s has best config %s", fn_hash, best_config)
```
Is a lazily evaluated string and way more efficient, particularly if the user does not want to actually log the data.